lib/exec.js

```diff
@@ -2,15 +2,17 @@ var spawn = require('child_process').spawn,
     objectAssign = require('object-assign');
 
 module.exports = function exec(script, program) {
+  script = (typeof script === 'string' ? {command: script} : script)
```

Comment:
It is preferred to assign a new variable here instead of reassigning a passed argument.